core/iwasm/fast-jit/cg/x86-64/jit_codegen_x86_64.cpp

```diff
@@ -2425,7 +2608,8 @@ static bool
 alu_imm_r_to_r_f32(x86::Assembler &a, ALU_OP op, int32 reg_no_dst,
                    float data1_src, int32 reg_no2_src)
 {
-    return false;
+    mov_imm_to_r_f32(a, reg_no_dst, data1_src);
```

Comment:
It is really a big problem. But since we create IR all by ourselves, it is possible to keep SSA in the front end and avoid such a drama. 

In one instruction, register allocation makes sure that it will not assign the same physical register to two virtual registers.

Reply:
my mistake about register allocation. it doesn't work in that way